src/orderbook.ts

```diff
@@ -1,6 +1,6 @@
 import {Order} from ".";
 
-export class Price {
+export class Fraction {
```

Comment:
Maybe this can be moved into its own file so all the rational number math is outside of the oderbook code.